CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
-## 2022-04-19 HFI Calc - Kelowna Station Additions [#1849](https://github.com/bcgov/wps/issues/1849)
+## 2022-04-19 HFI Calc
```

Comment:
Stupid stale date - I say don't worry about it? We really need an automated solution.